rllib/agents/dqn/dqn_torch_policy.py

```diff
@@ -244,6 +244,14 @@ def compute_q_values(policy, model, obs, explore, is_training=False):
     return q_values
 
 
+def grad_process_and_td_error_fn(policy, optimizer, loss):
```

Comment:
Added td_error key to info dict (needed by AsyncReplayOptim to update PR weights).